recipes/expat/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+    name: expat
+    version: 2.1.0
+
+source:
+    fn: expat-2.1.0.tar.gz
+    url: http://sourceforge.net/projects/expat/files/expat/2.1.0/expat-2.1.0.tar.gz/download
+
+build:
+    number: 0
+    skip: True # [not win]
```

Comment:
Do we need expat on Linux and OS X? `libexpat` is readily available on those platforms and we only use it for `udunits2` on Windows.